### ISSUE TITLE
Official IRC or Matrix

### ISSUE THREAD

**Issue by alien2003** [R2Northstar/Northstar#48]
I can see official Discord in description but not everyone agrees to trade their privacy for their favorite game so it would be great to also have official IRC or Matrix room, maybe bridged to Discord

**Comment by Agrigohar** [comment]
Well I think that is pretty ironic to say, because you are already trading your privacy with Microsoft because you use GitHub. And if you see what security flaws the Titanfall Software has, you might also trade stuff like your IP with anyone who has a bit of IT knowledge.

**Comment by iplayr5r** [comment]
>And if you see what security flaws the Titanfall Software has, you might also trade stuff like your IP with anyone who has a bit of IT knowledge.
VPN 🤦🏻

**Comment by iplayr5r** [comment]
>And if you see what security flaws the Titanfall Software has, you might also trade stuff like your IP with anyone who has a bit of IT knowledge.

VPN 🤦🏻

**Comment by elterrasense** [comment]
> 
> VPN 🤦🏻

Yeah because VPNs clearly work and they don't steal your data instead.
I could rant for hours about it and how no matter what you do nowadays there's always something spying on you, but it would be going overboard for just a github issue lol.
Anyways, having a bit of sense of privacy like this user is good, but it's no use being an schizo about it, at that point you might as well not use the internet at all.

**Comment by iplayr5r** [comment]
There are fully open-source VPNs which can be bought with crypto and don't need email, etc. Also there's an option of self hosting. I prefer to trust my VPN provider and don't get my IP exposed to some random out there.

**Comment by notbluu** [comment]
> 

this ain't GTA but I see your point.

Plus most people who play games wouldn't use IRC or Matrix anyways

**Comment by alien2003** [comment]
> Well I think that is pretty ironic to say, because you are already trading your privacy with Microsoft because you use GitHub. And if you see what security flaws the Titanfall Software has, you might also trade stuff like your IP with anyone who has a bit of IT knowledge.

Discord is known for crawling files and scanning process list on your computer. Titanfall 2 was not catched doing stuff like that.

**Comment by SIGSTACKFAULT** [comment]
> Discord is known for crawling files and scanning process list on your computer.

source?

**Comment by notbluu** [comment]
> > Well I think that is pretty ironic to say, because you are already trading your privacy with Microsoft because you use GitHub. And if you see what security flaws the Titanfall Software has, you might also trade stuff like your IP with anyone who has a bit of IT knowledge.

Sources?

**Comment by alien2003** [comment]
> source?

https://spyware.neocities.org/articles/discord.html

If you don't trust this source, you can do your own research by running gdb~W Process Monitor

**Comment by notbluu** [comment]
>neocities

**Comment by GeckoEidechse** [comment]
Fuck it, I just made one now on matrix.org :P
https://matrix.to/#/#r2northstar:matrix.org

**Comment by zImPatrick** [comment]
> Discord is known for crawling files and scanning process list on your computer. Titanfall 2 was not catched doing stuff like that. Github also can't access data on your computer

Couldn't you just use discord web?

**Comment by alien2003** [comment]
> Couldn't you just use discord web?

It still requires phone number to use, not good for FOSS projects
